python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -8,7 +8,7 @@
 
 import torch
 
-from ...base import ETYPE
+from ...base import EID, ETYPE
```

Comment:
Graphbolt should not have dependency on DGL graph's EID definition.

Note: the ETYPE was added for converting DGL graph to csc_sampling_graph.